src/main/kotlin/RedstoneTools.kt

```diff
@@ -55,6 +57,8 @@ class RedstoneTools : JavaPlugin() {
             WorldEditHelper(this, worldEdit),
             SlabListener(),
             autowire,
+            AutoRotate(),
```

Comment:
These need to be in a variable (like autowire), so that there's just one instance that gets registered as a command and listener. The ones that are directly instantiated in the lists are either just listeners or just commands but not both.